indexer/src/main.ts

```diff
@@ -66,13 +79,21 @@ export const config: Config<NetworkOptions, SinkOptions> = {
 export default async function transform({
   header,
   events,
+  transactions,
 }: {
   header: BlockHeader;
   events: EventWithTransaction[];
+  transactions: TransactionWithReceipt[];
 }) {
   // Accumulate the gas used in the block in order to calculate the cumulative gas used.
   // We increment it by the gas used in each transaction in the flatMap iteration.
   let cumulativeGasUsed = 0n;
+  // The cumulative gas uses is an array containing the transaction index and the
+  // cumulative gas used up to that transaction. This is used to later
+  // get the cumulative gas used for a out of resources transaction.
+  const cumulativeGasUses: Array<{ index: number; cumulativeGasUsed: bigint }> =
```

Comment:
this variable name is confusing, why not a plain array leaving `empty` some indexes?

Reply:
that works actually, will update